blueprint/api.apib

```diff
@@ -2795,6 +2821,18 @@ The platform stores Stripe customers and cards so they can be reused across diff
     + title: `404 Resource not found` (string)
     + status: 404 (number) - HTTP status code
 
+## Reset Password Resource (object)
++ include Reset Password Resource Identifier
+
+## Reset Password Resource Identifier
++ token: `zzz123` (string, required)
++ password: `mypwd` (string, required)
++ password-confirmation: `mypwd` (string, required)
```

Comment:
I'd just move this under `Reset Password Resource`. If the identifier and the resource are the same, there's little point of having the identifier. Just gives an extra step in reading the file.